tests/conftest.py

```diff
@@ -295,7 +243,7 @@ def df_type(request: pytest.FixtureRequest):
         use_cudf = request.node.get_closest_marker("use_cudf") is not None
 
         if (use_pandas and use_cudf):
-            raise RuntimeError(f"Both markers (use_cpp and use_python) were added to function {request.node.nodeid}. "
+            raise RuntimeError(f"Both markers (gpu_mode and cpu_mode) were added to function {request.node.nodeid}. "
```

Comment:
I think this is an error. Shouldnt it be `use_pandas` and `use_cudf`? The original was wrong too

Reply:
Good catch, I think I just did a find/replace here